commander_unix.yaml

```diff
@@ -128,10 +128,17 @@ tests:
     config:
       env:
         USER: from_parent
-    command: ./commander test https://raw.githubusercontent.com/commander-cli/commander/master/integration/unix/commander_test.yaml
+    command: ./commander test https://raw.githubusercontent.com/commander-cli/commander/master/integration/unix/remote_http.yaml
     stdout:
       contains:
-        - ✓ [local] it should exit with error code
-        - "- [local] it should skip, was skipped"
-      line-count: 17
-    exit-code: 0
\ No newline at end of file
+        - ✓ [local] hello world
+    exit-code: 0
+
+  test retries with stdin:
```

Comment:
See at my comment for the nodes and stdin test.

Adding a test to `unix` is enough because the build target `intergration-linux` already executes all this `unix` tests.